vector-im/element-android
# Removing `Login2` proof of concept

## Type of change


## Content

Removes all `Login2` related classes and resources, this flow has been superseded by the FTUE changes

## Motivation and context

#5974 Removes the no longer needed Login2 PoC

## Screenshots / GIFs
No UI changes

## Tests

- open the debug menu
- notice Login2 is no longer an option

## Tested devices

- OS version(s):
